### PR TITLE
Fix: broken links in 3.8

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description
Update semantic cache links to new name and add missing AI plugins links.

### Testing instructions

Link test should pass.
### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_hub/kong-inc/ai-semantic-cache/overview/_index.md [hunk 1]
```diff
@@ -110,9 +110,9 @@ The plugin respects cache control headers to determine if requests and responses
 
 ## Get started with the AI Semantic Caching plugin
 
-* [Configuration reference](/hub/kong-inc/ai-semantic-caching/configuration/)
-* [Basic configuration example](/hub/kong-inc/ai-semantic-caching/how-to/basic-example/)
-* [Learn how to use the plugin](/hub/kong-inc/ai-semantic-caching/how-to/)
+* [Configuration reference](/hub/kong-inc/ai-semantic-cache/configuration/)
+* [Basic configuration example](/hub/kong-inc/ai-semantic-cache/how-to/basic-example/)
+* [Learn how to use the plugin](/hub/kong-inc/ai-semantic-cache/how-to/)
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/ai-semantic-cache/overview/_index.md:115]:
```suggestion
* [Learn how to use the plugin](/hub/kong-inc/ai-semantic-cache/how-to/basic-example/)
```
I think that's the right link, the other gave me a 404 still

File: app/_includes/md/ai-plugins-links.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 * [AI Proxy](/hub/kong-inc/ai-proxy/)
+{% if_version gte:3.8.x -%} 
+* [AI Proxy Advanced](/hub/kong-inc/ai-proxy-advanced/) <span class="badge enterprise"></span>
+{% endif_version -%}
 * [AI Request Transformer](/hub/kong-inc/ai-request-transformer/)
 * [AI Response Transformer](/hub/kong-inc/ai-response-transformer/)
 {% if_version gte:3.8.x -%} 
-* [AI Semantic Caching](/hub/kong-inc/ai-semantic-caching/) <span class="badge enterprise"></span>
+* [AI Semantic Caching](/hub/kong-inc/ai-semantic-cache/) <span class="badge enterprise"></span>
```
Comment (cloudjumpercat) [inline app/_includes/md/ai-plugins-links.md:8]:
```suggestion
* [AI Semantic Cache](/hub/kong-inc/ai-semantic-cache/) <span class="badge enterprise"></span>
```

File: app/_hub/kong-inc/ai-semantic-cache/overview/_index.md (hunk 1 above)
Comment (lena-larionova) [reply app/_hub/kong-inc/ai-semantic-cache/overview/_index.md:115]: Hmm. This is just a page that doesn't exist at all.

Comment (lena-larionova) [inline app/_hub/kong-inc/ai-semantic-cache/overview/_index.md:115]:
```suggestion
* [Set up AI Semantic Cache with Mistral]/hub/kong-inc/ai-semantic-cache/how-to/llm-provider-integration-guides/mistral/)
* [Set up AI Semantic Cache with OpenAI]/hub/kong-inc/ai-semantic-cache/how-to/llm-provider-integration-guides/openai/)
```

**Pull request merged**